django_socio_grpc/services/servicer_proxy.py

```diff
@@ -230,33 +247,44 @@ def get_handler(self, action: str):
         return self._get_handler(action)
 
     def create_service(self, **kwargs):
-        return self.service_class(**self.initkwargs, **kwargs)
+        service = self.service_class(**self.initkwargs, **kwargs)
+        Ctx.service = service
+        return service
 
     def __getattr__(self, action):
         if not hasattr(self.service_class, action):
             raise Unimplemented()
 
         return self.get_handler(action)
 
-    def process_exception(self, exc, request: GRPCRequestContainer):
+    def process_exception(self, exc, request_container: GRPCRequestContainer):
         if isinstance(exc, GRPCException):
-            logger.error(exc)
-            request.context.abort(exc.status_code, exc.get_full_details())
-        elif isinstance(exc, grpc.RpcError) or request.context._state.aborted:
+            logger.error(
+                exc.detail,
```

Comment:
```suggestion
                f"{type(exc).__name__} : {exc}",
```
after testing, with .detail there's only the exception message